docs/getting_started.md

```diff
@@ -176,3 +227,20 @@ To send test run reports to email ID's, postfix should be installed on fedora
     * `sudo dnf install postfix`
     * `systemctl enable postfix.service`
     * `systemctl start postfix.service`
+
+
+## Notes
+
+[1]: For Mac OSX Users
```

Comment:
just maybe create the section for it? You can have a look at preview of file here:  https://github.com/red-hat-storage/ocs-ci/blob/7ea00803efa1612c3484eae00f0df054a243e7eb/docs/getting_started.md. This doesn't look good to me and will look better if it will be in own section and you can also have that anchor link to it and not use extra indention here ;) .